http/servlet-undertow/src/test/java/io/quarkus/ts/http/undertow/WithMissingServlet.java

```diff
@@ -0,0 +1,14 @@
+package io.quarkus.ts.http.undertow;
+
+import static io.quarkus.ts.http.undertow.UndertowMissingServletClassIT.replaceForInvalidXML;
+
+import io.quarkus.test.services.quarkus.ProdQuarkusApplicationManagedResourceBuilder;
+
+public class WithMissingServlet extends ProdQuarkusApplicationManagedResourceBuilder {
```

Comment:
Maybe this should had been `WithInvalidServlet` or whatever, but it doesn't really matter.